projects/smarti18n/cli/commandBuild.js

```diff
@@ -133,16 +175,34 @@ module.exports = class BuildCommand {
 	 */
 	getSrcPath(project = undefined) {
 		const angularConfigPath = path.join(process.cwd(), 'angular.json');
-		if (!fs.existsSync(angularConfigPath))
+		if (!fs.existsSync(angularConfigPath)) {
+			console.log(chalk.red('error: Angular config not found!'));
 			throw new Error('Angular config not found');
+		}
+
 		const angularConfig = require(angularConfigPath);
-		project = project || angularConfig.defaultProject;
+		project = project || angularConfig._DEFAULTPROJECT;

```

Comment:
this is an angular-cli default, should not change the name of the property from `defaultProject`

Reply:
fixed